core/src/components/input/input.scss

```diff
@@ -202,11 +203,11 @@
 
   color: $text-color-step-400;
 
-  visibility: hidden;
   appearance: none;
 }
 
-:host(.in-item-color) .input-clear-icon {
+:host(.in-item-color) .input-clear-icon,
+:host(.in-item-color) .input-password-toggle {
```

Comment:
let's break this into

```
:host(.in-item-color){
    .input-clear-icon,
    .input-password-toggle {  
    } 
}
```

Reply:
I would like to keep it as it is, because it is more readable and also matches the current CSS structure of the code.

Reply:
@sean-perkins what do you think about this topic?

Reply:
Most CSS we've written in Ionic avoids nested to keep the code readable. However, I think either would be fine -- really comes down to a matter of preference.